### PR TITLE
Some refactoring, switches to using icons, adds tooltips, adds Grunt, removes auto .min hiding, works on github.com now.

### PR DESCRIPTION
**Pull request by himedlooff**
Also automatically loads for pull request pages now too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,9 +27,11 @@ $('[data-path*=".min"]').parent().hide();
...
 
 Then in `manifest.json` update the `matches` value under `content_scripts`.
-Since the jQuery selectors are geared for GHE you'd want to use a matching
-pattern like fake-ghe-domain/*/*/compare/* so the script only runs on compare
-view pages.
+Since the jQuery selectors are geared for GHE you'd want to update the patterns
+provided to point to your GitHub Enterprise domain.
+Currently they target comparison and pull request pages.
+More on tweaking the matching patterns can be found
+[here]("http://developer.chrome.com/extensions/content_scripts#match-patterns-globs").
```
Comment (Scotchester) [inline README.md]: These quotes prevent this link from working.

**Comment by Scotchester** [comment]
Is there a reason for checking in the Bower components?

**Comment by Scotchester** [comment]
I guess if you didn't people would have to run `bower install` before installing the extension. NM.

**Comment by himedlooff** [comment]
Yeah... so maybe it's worth setting up grunt-bower-task to pull in only what we need then. This would lighten up the amount of files here. I think I'll do that to make this PR slimmer.

File: README.md (hunk 1 above)
Comment (himedlooff) [inline README.md]: Thanks, fixed.

**Comment by Scotchester** [comment]
So uh... just found out about this: https://github.com/defunkt/dotjs

Should we consider just setting this up to plug into that?

**Comment by himedlooff** [comment]
It would be pretty trivial to do both since whatever is in app.js can simply be placed in a dotjs folder. 

Since the Chrome extension is already working we should just stick with it. It's still useful for user triggered actions, like if we want to allow them to click the extension icon in the browser chrome and check off a few options for the current page. Also it's kind of fun to play around with.

**Comment by Scotchester** [comment]
Sounds good! :+1:

**Comment by Scotchester** [comment]
Hey, you going to do the `grunt-bower-task` setup?

**Comment by himedlooff** [comment]
yeah i should, forgot about it.

**Comment by himedlooff** [comment]
Updated!

File: README.md [hunk 2]
```diff
@@ -24,14 +24,26 @@ $('[data-path*=".min"]').parent().hide();
...
-Then in `manifest.json` update the `matches` value under `content_scripts`.
-Since the jQuery selectors are geared for GHE you'd want to use a matching
-pattern like fake-ghe-domain/*/*/compare/* so the script only runs on compare
-view pages.
-
-Follow these [instructions](http://developer.chrome.com/extensions/getstarted#unpacked)
+### Requirements
+
+- [npm](https://npmjs.org/)
+- [grunt-cli](http://gruntjs.com/getting-started)
```
Comment (Scotchester) [inline README.md:47]: Need to add Bower (installed globally) to this list.

Comment (himedlooff) [inline README.md:47]: Added.

**Comment by Scotchester** [comment]
Should we be committing the `vendor` folder? And the links in `manifest.json` should point to `vendor`, not `bower_components`, right?

**Comment by himedlooff** [comment]
Oops, yeah I updated manifest.json but not mainfest-sample.json. Will do that now.

**Comment by himedlooff** [comment]
`vendor` is getting committed.

**Comment by himedlooff** [comment]
Updated the instructions.

**Comment by himedlooff** [comment]
Updated.

**Comment by Scotchester** [comment]
Awesome. Love it. Great work! :shipit:

**Pull request merged**